### PR TITLE
[DOCFIX] Add javadoc for PinIntegrationTest

### PR DESCRIPTION
**Pull request by swatDong**

### CODE REVIEW

File: tests/src/test/java/alluxio/master/PinIntegrationTest.java [hunk 1]
```diff
@@ -34,6 +34,9 @@
 import java.net.InetSocketAddress;
 import java.util.HashSet;
 
+/**
+ * Test the pin operation of file and directory.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/PinIntegrationTest.java]: Maybe just "Integration tests for the pin operation"? I don't see any reason to call out the file and directory cases specifically

File: tests/src/test/java/alluxio/master/PinIntegrationTest.java [hunk 2]
```diff
@@ -58,6 +61,10 @@ public final void after() throws Exception {
     mFSMasterClient.close();
   }
 
+  /**
+   * Test recursive pinness that pinning/unpinning a folder should recursively take effect on its
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/PinIntegrationTest.java]: Tests that pinning/unpinning ...

File: tests/src/test/java/alluxio/master/PinIntegrationTest.java [hunk 3]
```diff
@@ -100,10 +107,12 @@ public void recursivePinness() throws Exception {
         Sets.newHashSet(status.getFileId()));
   }
 
+  /**
+   * Test inherit pinness that children should inherit the isPinned attribute from their parent
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/PinIntegrationTest.java]: Tests that children should ...

**Comment by swatDong** [comment]
Thanks for comments. I've updated.

**Pull request merged**